src/language-css/options.js

```diff
@@ -3,6 +3,15 @@
 const commonOptions = require("../common/common-options");
 
 // format based on https://github.com/prettier/prettier/blob/main/src/main/core-options.js
+/** @type {{ [name: string]: import("../main/core-options").OptionInfo }} */
```

Comment:
ditto: I think it would be ideal to propose changes like this in the upstream Prettier project.